Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -1022,7 +1022,9 @@ GDCMImageIO::Write(const void * buffer)
   if (hasIPP)
   {
     double origin3D[3];
+    std::locale currentLocale = std::locale::global(std::locale::classic());//save and reset old locale
```

Comment:
```suggestion
    std::locale currentLocale = std::locale::global(std::locale::classic()); // save and reset old locale
```

Reply:
I would like to commit your suggestions, but the pre commit hook does not let me do it. Got any tip to apply?
```
[attr]our-c-style  whitespace=tab-in-indent,no-lf-at-eof  hooks.style=KWStyle,clangformat
 not allowed: ./.gitattributes:31
[attr]our-c-style  whitespace=tab-in-indent,no-lf-at-eof  hooks.style=KWStyle,clangformat
 not allowed: ./.gitattributes:31
Modules/IO/GDCM/src/itkGDCMImageIO.cxx:69: error: Unnecessary semicolon
Modules/IO/GDCM/src/itkGDCMImageIO.cxx:156: error: Unnecessary semicolon
Modules/IO/GDCM/src/itkGDCMImageIO.cxx:245: error: Unnecessary semicolon
pre-commit hook failure
-----------------------

KWStyle check failed.

Line numbers in the errors shown refer to the file:
Modules/IO/GDCM/src/itkGDCMImageIO.cxx.kws

```

Reply:
We could put the comment above the code line.

Reply:
I couldn't commit because of KWStyle throwing an bogus error on a line I did not modify. Seems to be somewhat related to this discussion [https://discourse.itk.org/t/help-with-bogus-pre-commit-hook-failure/2350/7](https://discourse.itk.org/t/help-with-bogus-pre-commit-hook-failure/2350/7) . Though I made sure that I had a fresh build with the KWStyle binary in place. I had to disable kwstyle checking...